src/recipeUtils.js

```diff
@@ -0,0 +1,15 @@
+export function recipesFromTag(recipes, tag) {
+  return recipes.filter((recipe) => {
+    return recipe.tags.includes(tag)
+  });
+}
+
+export function recipesfromName(recipes, name) {
+  return recipes.filter((recipe) => {
+    return recipe.name.includes(name)
+  })
+}
```

Comment:
Single responsibility functions! Very nice!

Reply:
I love that you ended up using includes! That really helps with the error handling!